chef/cookbooks/rabbitmq/recipes/rabbit.rb

```diff
@@ -81,7 +81,10 @@
   only_if only_if_command if ha_enabled
 end
 
-if node[:rabbitmq][:trove][:enabled]
```

Comment:
@rhafer ping? (I thought you replied a while ago, but don't see the comment here)